modules/rules_swift/2.2.0/presubmit.yml

```diff
@@ -0,0 +1,37 @@
+shell_commands: &shell_commands
+- "echo --- Downloading and extracting Swift $SWIFT_VERSION to $SWIFT_HOME"
+- "mkdir $SWIFT_HOME"
+- "curl https://download.swift.org/swift-${SWIFT_VERSION}-release/ubuntu2004/swift-${SWIFT_VERSION}-RELEASE/swift-${SWIFT_VERSION}-RELEASE-ubuntu20.04.tar.gz | tar xvz --strip-components=1 -C $SWIFT_HOME"
+
+matrix:
+  bazel:
+  - 7.x
+  - last_green
```

Comment:
```suggestion
#  - last_green
```

Reply:
The closest I can do to a force merge

Reply:
I don't think we should specify last_green here, how about 8.0.0rc1? And if that doesn't work, we should remove it and keep the presubmit green.

Reply:
Pretty sure 8.0.0rc1 fails as well. I'll check, and change or remove it.

Reply:
Removed altogether for now.